host/sgx/sgxquoteprovider.c

```diff
@@ -92,7 +92,7 @@ oe_result_t oe_get_sgx_quote_verification_collateral(
     }
 
     if (collateral->version != SGX_QL_QVE_COLLATERAL_VERSION_1 &&
-        collateral->version != SGX_QL_QVE_COLLATERAL_VERSION_3)
```

Comment:
What happen if the version is `SGX_QL_QVE_COLLATERAL_VERSION_3` with this change?

Reply:
Don't drop support for version 3.0.

Reply:
These changes seem insufficient. The format of CRL is changing with 3.1. Please consider changing CRL reading code too, else failure will shift from this point to later in code where CRL is being interpreted.

Reply:
`OE` is able to handle both PEM CRL and DER CRL, but it's not able to handle hex string CRL. For version < 3.0, CRL is a PRM, for version > 3.0 (e.g. 3.1), CRL is a DER. For version == 3.0, SGX returns a hex string that OE is not able to handle. But Azure DCAP client somehow is able to store a DER CRL for 3.0 collateral as is now so now OE is compatible with a V3 Azure DCAP collateral. Is Azure DCAP client going to retrieve a hex string CRL for 3.0 collateral in the future @Shivr-ctrl?

Reply:
Note that #4272 adds support for hex CRL

Reply:
Adopted changes from PR #4272, to support hex CRLs